runsharp/full_gui.py

```diff
@@ -2,12 +2,31 @@
 import numpy as np
 import warnings
 import utils.frozenutils as frozenutils
+import logging
 
 HOME_DIR = os.path.join(os.path.expanduser("~"), ".sharppy")
 
 if len(sys.argv) > 1 and '--debug' in sys.argv:
     debug = True
     sys.path.insert(0, os.path.normpath(os.getcwd() + "/.."))
+
+    # Start the logging
+    logging.basicConfig(level=logging.DEBUG,
+                        format='%(asctime)s %(pathname)s %(funcName)s Line #: %(lineno)d %(levelname)-8s %(message)s',
+                        filename='sharppy.log',
+                        filemode='w')
+    # define a Handler which writes INFO messages or higher to the sys.stderr
+    console = logging.StreamHandler()
+    console.setLevel(logging.DEBUG)
+    # set a format which is simpler for console use
+    formatter = logging.Formatter('%(name)-12s: %(levelname)-8s %(message)s')
+    # tell the handler to use this format
+    console.setFormatter(formatter)
+    # add the handler to the root logger
+    logging.getLogger('').addHandler(console)
+
+    logging.info('Started logging output for SHARPpy')
+
```

Comment:
So this section looks like it sets up logging only if the --debug flag is passed. Should we set up some stuff for the non-debug version, too? The level can be logging.INFO, which will ignore all the debug statements.

Reply:
We can do that.  I only set it up because I was thinking about using it for our debugging (or others) of the code.  Otherwise running it normally I didn't want the command prompt to be really noisy that people would think something was wrong.